applications/nrf5340_audio/doc/configuration.rst

```diff
@@ -0,0 +1,63 @@
+.. _nrf53_audio_app_configuration:
+
+Configuring the nRF5340 applications
+####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|config|
+
+.. _nrf53_audio_app_configuration_select_bidirectional:
+
+Selecting the CIS bidirectional communication
+*********************************************
+
+The CIS unidirectional mode is the default operating mode for the application.
+To switch to the bidirectional mode, set the :ref:`CONFIG_STREAM_BIDIRECTIONAL <config_nrf53_audio_app_options>` Kconfig option to ``y``  in the :file:`prj.conf` file (for the debug version) or in the :file:`prj_release.conf` file (for the release version).
+
+.. _nrf53_audio_app_configuration_enable_walkie_talkie:
+
+Enabling the walkie-talkie demo
+===============================
+
+The walkie-talkie demo uses one or two bidirectional streams from the gateway to one or two headsets.
+The PDM microphone is used as input on both the gateway and headset device.
+To switch to using the walkie-talkie, set the :ref:`CONFIG_WALKIE_TALKIE_DEMO <config_nrf53_audio_app_options>` Kconfig option to ``y``  in the :file:`prj.conf` file (for the debug version) or in the :file:`prj_release.conf` file (for the release version).
+
+.. _nrf53_audio_app_configuration_select_i2s:
+
+Selecting the I2S serial
+************************
+
+In the default configuration, the gateway application uses the USB serial port as the audio source.
+The :ref:`nrf53_audio_app_building` and :ref:`nrf53_audio_app_testing` steps also refer to using the USB serial connection.
```

Comment:
I guess these refs are invalid now? Would probably be the same for most of the refs starting with nrf53_audio_app_ in this file

Reply:
These are still valid, pointing to pages in the doc/ for the app.